src/renderer/components/Editor/UngroupTakesModal.tsx

```diff
@@ -0,0 +1,77 @@
+import { Modal, styled, Box, Button, Typography, Stack } from '@mui/material';
+import colors from 'renderer/colors';
+
+const CustomStack = styled(Stack)`
+  width: 100%;
+  height: 100%;
+`;
+
+const CustomModal = styled(Modal)`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+`;
+
+const CustomModalInner = styled(Box)`
+  background: ${colors.grey[700]};
+  padding: 15px 30px 30px 30px;
+  border-radius: 5px;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  width: 420px;
+`;
+
+const CustomRowStack = styled(CustomStack)`
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+`;
+
+const CustomButton = styled(Button)`
+  filter: drop-shadow(0px 8px 16px rgba(0, 0, 0, 0.8));
+  width: 100%;
+`;
+
+interface Props {
+  isOpen: boolean;
+  closeModal: () => void;
+  ungroupTakes: () => void;
+}
+
+const UngroupTakesModal = ({ isOpen, closeModal, ungroupTakes }: Props) => {
+  const ungroupTakesButton = (
+    <CustomButton color="primary" onClick={ungroupTakes}>
+      Ungroup Takes
+    </CustomButton>
+  );
+
+  const cancelButton = (
+    <CustomButton color="secondary" onClick={closeModal}>
+      Cancel
+    </CustomButton>
+  );
+
+  return (
+    <div>
+      <CustomModal open={isOpen} onClose={closeModal}>
+        <CustomModalInner>
+          <CustomStack>
+            <Typography variant="h1" color={colors.grey[400]}>
+              Ungroup Takes
+            </Typography>
+            <Typography variant="p-300" paddingTop="10px">
+              Are you sure you want to ungroup takes? This is not reversible.
```

Comment:
Why shouldn't it be reversible? We can easily bake it into undo/redo. In fact if we don't, it kind of breaks the whole undo/redo system - you can't really have "some stuff is reversible and other things aren't". Even if the only way to get the takes back is by undoing, I think this message is misleading

Reply:
The original discussion when producing these UI designs for take detection was that it might be too difficult to undo the ungrouping of takes. Idk personally if that's the case. @emmapainter thoughts?

Reply:
Nah it's not difficult at all. Although I guess it also won't *really* break undo if we don't make it undoable since take state is kind of orthogonal to the rest of the editor state, but the invariant of "should be able to restore a project to its original state" will be gone

Reply:
If it's not difficult then I'd love for it to be reversible - thanks for pulling this up @chloebrett, probably something we should have investigated before writing it off as too difficult.

Reply:
Sounds good, I've just started trying to implement this. Should I tack it on to this PR or to a new one?

Reply:
Lol I just refreshed the page and saw Emma and your comments